test/dumps/module_data.json

```diff
@@ -0,0 +1 @@
+"key1":"<<BINARY>>"
```

Comment:
maybe we can print the module name and maybe size of the payload?

Reply:
Maybe we can optionally return some kind of iterator (for int, float, double, double & string) and let handlers to decide what they want to do with it ... But then the handlers might need to deal with `RDB_STATUS_WAIT_MORE_DATA`. If so, then it will be simpler to have another group of callbacks to submodules (that called only after each item was entirely read) and let the handlers do the math, if it wants the size.

Reply:
IIUC, you're suggesting that users can implement extensions to the parser that can decode module keys and translate them to usable content (e.g. for ReJson, they'll be able to print the JSON content of the key).
that sounds like a good idea, maybe we can add that in a todo and handle it in a second phase.

meanwhile, for someone that dumps an RDB into json and some keys are module keys, i think just saying "BINARY" is insufficient, and i'd like to state the module name and maybe the payload size.